app/socket-apis/__tests__/get-user-whys.js

```diff
@@ -0,0 +1,95 @@
+// https://github.com/EnCiv/civil-pursuit/issues/206
+
+const { MongoMemoryServer } = require('mongodb-memory-server')
+const { MongoClient } = require('mongodb')
+const getUserWhys = require('../get-user-whys')
+const Points = require('../../models/points')
+
+let memoryServer
+let connection
+let db
+const userId = '12345678abcdefgh'
+const synuser = { synuser: { id: userId } }
+
+beforeAll(async () => {
+  memoryServer = await MongoMemoryServer.create()
+  const uri = memoryServer.getUri()
+  connection = await MongoClient.connect(uri, { useNewUrlParser: true })
+  db = connection.db()
+  Points.find = jest.fn().mockImplementation(query => ({
```

Comment:
Rather than mock the find, please insert data into the database and then find it. It really helps find problems to be able to test the APIs all the way through into the database. And the mongodb memory server makes this really quick and easy.